packages/sdk-components-react/src/html-embed.test.tsx

```diff
@@ -0,0 +1,359 @@
+/**
+ * @jest-environment jsdom
+ */
+import * as React from "react";
+// import ReactDOM from "react-dom";
+import ReactDOMServer from "react-dom/server";
+import { test, expect, describe } from "@jest/globals";
+// eslint-disable-next-line import/no-internal-modules
+import "@testing-library/jest-dom/jest-globals";
+import { fireEvent, render, screen } from "@testing-library/react";
+import { SCRIPT_PROCESSED_TEST_ID_PREFIX, HtmlEmbed } from "./html-embed";
+import { ReactSdkContext } from "@webstudio-is/react-sdk";
+
+global.React = React;
```

Comment:
that shouldn't be here, either lets add @todo to remove later with playwright rewrite or we need this centralized